classes/EbaySynchronizer.php

```diff
@@ -533,21 +539,29 @@ private static function _hasVariationsMatching($product_id, $id_lang, $ebay_cate
 
 	private static function _getPrices($product_id, $percent, $ebay_profile)
 	{
-		$price = Product::getPriceStatic((int)$product_id, true);
-		$price_original = Product::getPriceStatic((int)$product_id, true, null, 6, null, false, false);
+		$context = clone Context::getContext();
+		$context->shop = new Shop($ebay_profile->id_shop);
 		
-		// convert price to destination currency
-		$currency = new Currency($ebay_profile->getConfiguration('EBAY_CURRENCY'));
-		$price *= $currency->conversion_rate;
-		$price_original *= $currency->conversion_rate;
+		$specific_price_output = null;
+
+		$price = Product::getPriceStatic((int)$product_id, true, null, 6, null, false, true, 1, false, null, null, null, $specific_price_output, true, true, $context);
 		
+		$specific_price_output = null;

```

Comment:
I do not understand this part. We have a double affectation of the same value in a single variable.
It seems to be better to write `null` directly in the function parameters.